exercises/all-your-base/instructions.md

```diff
@@ -26,8 +23,6 @@ The number 1120, _in base 3_, means:
 
 `(1 * 3^3) + (1 * 3^2) + (2 * 3^1) + (0 * 3^0)`
 
-I think you got the idea!
-
```

Comment:
We should not be implying that students should understand something, because if they don't, they could feel bad about themselves.